docs/querying/set-query-context.md

```diff
@@ -0,0 +1,240 @@
+---
+id: set-query-context
+title: "Set query context"
+sidebar_label: "Set query context"
+---
```

Comment:
Suggest inclduing a `description:` front matter that summarizes the document. Not widely used in the Druid docs right now, but it is helpful with search results.